playwright-e2e/tests/smoke.spec.ts

```diff
@@ -78,6 +78,12 @@ test.describe('smoke test', () => {
     await app.reviewSubmit.statementOfTruthOne(appOneName.appOneFullname);
     await app.reviewSubmit.fillCardDetails(appOneName.appOneFullname, userEmail, postcode1);
 
+    // const caseIdLocator = app.page.locator('.govuk-panel__body strong');
+    const element = await app.page.locator('.govuk-panel.govuk-panel--confirmation .govuk-panel__body strong').first();
+    const someText = await element.textContent();
+    console.log(someText);
+    await app.page.pause();
```

Comment:
Remove the pause() , it might be added accidentally

Reply:
redundant code removed